distributions/LibreELEC/version

```diff
@@ -5,4 +5,4 @@
   OS_VERSION="12.2"
 
 # ADDON_VERSION: Addon version
-  ADDON_VERSION="12.0.0"
+  ADDON_VERSION="12.2.0"
```

Comment:
shouldn't that be 12.1.0 so that we can use 12.2.0 in the official 12.2.0 release?